RTDiag/Core/Src/MainAPP/NVMM/NVMM.c

```diff
@@ -1,8 +1,17 @@
-/*
- * NVMM.c
- *
- *  Created on: Jan 31, 2024
- *      Author: Hamdi
- */
+#include "stdint.h"
+#include "stm32f4xx_hal.h"
+#include "string.h"
 
+void NV_save_data(uint32_t Address,uint32_t data){
```

Comment:
Arguments should be user friendly (enum names, macros, .. )
this function can be static